server/admin.go

```diff
@@ -49,6 +50,8 @@ const (
 	debugEndpoint = "/debug/"
 	// healthzPath is the healthz endpoint.
 	healthzPath = adminEndpoint + "healthz"
+	// quitPath is the quit endpoint.
+	quitPath = adminEndpoint + "quitquitquit"
```

Comment:
We don't have to follow google's naming conventions here; everything is under `/_admin/` so we can keep the last component of the path simple without risk of collision. I'd just call this "quit" (and for that matter remove the z from "healthz").

Before we get to production status we'll also want to make any HTTP-accessible quit handler xsrf-protected instead of a simple get (and maybe just remove it completely or make it a development-only feature).